contracts/0.4.24/Lido.sol

```diff
@@ -111,151 +83,165 @@ contract Lido is ILido, StETH, AragonApp {
     /// Not used in the logic
     bytes32 internal constant TOTAL_EL_REWARDS_COLLECTED_POSITION = keccak256("lido.Lido.totalELRewardsCollected");
 
-    /// @dev Credentials which allows the DAO to withdraw Ether on the 2.0 side
-    bytes32 internal constant WITHDRAWAL_CREDENTIALS_POSITION = keccak256("lido.Lido.withdrawalCredentials");
+    /// @dev version of contract
+    bytes32 internal constant CONTRACT_VERSION_POSITION = keccak256("lido.Lido.contractVersion");
 
     /**
-    * @dev As AragonApp, Lido contract must be initialized with following variables:
-    * @param _depositContract official ETH2 Deposit contract
-    * @param _oracle oracle contract
-    * @param _operators instance of Node Operators Registry
-    * @param _treasury treasury contract
-    * @param _insuranceFund insurance fund contract
-    * NB: by default, staking and the whole Lido pool are in paused state
-    */
+     * @dev As AragonApp, Lido contract must be initialized with following variables:
+     * @param _oracle oracle contract
+     * @param _treasury treasury contract
+     * @param _stakingRouterAddress Staking router contract
+     * @param _dsmAddress Deposit security module contract
+     * NB: by default, staking and the whole Lido pool are in paused state
+     */
     function initialize(
-        IDepositContract _depositContract,
-        address _oracle,
-        INodeOperatorsRegistry _operators,
-        address _treasury,
-        address _insuranceFund
-    )
-        public onlyInit
-    {
-        NODE_OPERATORS_REGISTRY_POSITION.setStorageAddress(address(_operators));
-        DEPOSIT_CONTRACT_POSITION.setStorageAddress(address(_depositContract));
+        address _oracle, 
```

Comment:
there are multiple trailing spaces here and throughout the contract code and comments